### DISCUSSION TITLE
change buymeacoffee to ko-fi

### DISCUSSION THREAD

**Discussion by AegisBlue** [nunocoracao/blowfish#594] [Q&A] [answered]
Hello,

I want to change the buymeacoffee widget + settings to ko-fi, does anyone know where the file is located?

Thanks!

**Comment by nunocoracao** [comment, marked as answer]
Here it is https://github.com/nunocoracao/blowfish/blob/789ec7068faea1abc3f639bc6b78b74756798277/layouts/_default/baseof.html#L38

not a seperate file at this stage. But I would say it doesn't need to be. You can simply not configure buymeacoffee and extend the header/footer to configure ko-fi.

**Reply by AegisBlue** [reply]
Thank you!

Works perfectly.
